spec/factories.rb

```diff
@@ -1,6 +1,18 @@
 # frozen_string_literal: true
 
 FactoryBot.define do
+  factory :user do
```

Comment:
let's move each factory into separate file. So when you start adding traits it won't be mess

Reply:
yes I think it will be better. I sort of wanted to do it later, but it flew out of my head)